### PR TITLE
HRINT-356 Java Demos - Auto Indexes

### PR DESCRIPTION
**Pull request by Danielle9897**

### CODE REVIEW

File: DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/demos.json [hunk 1]
```diff
@@ -59,6 +59,17 @@
             "PagingQueryResults"
         ]
     },
+    {
+        "category": {
+            "slug": "auto-indexes",
+            "directory": "autoIndexes"
+        },
+        "demos": [
+            "AutoMapIndex1",
```
Comment (topalach) [inline DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/demos.json]:
```suggestion
            "autoMapIndex1",
```

File: DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/demos.json [hunk 2]
```diff
@@ -59,6 +59,17 @@
...
+            "AutoMapIndex2",
```
Comment (topalach) [inline DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/demos.json]:
```suggestion
            "autoMapIndex2",
```

File: DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/demos.json [hunk 3]
```diff
@@ -59,6 +59,17 @@
...
+            "AutoMapReduceIndex"
```
Comment (topalach) [inline DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/demos.json]:
```suggestion
            "autoMapReduceIndex"
```

File: DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/demos.json (hunk 1 above)
Comment (Danielle9897) [reply DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/demos.json]: done

File: DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/demos.json (hunk 2 above)
Comment (Danielle9897) [reply DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/demos.json]: done

File: DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/demos.json (hunk 3 above)
Comment (Danielle9897) [reply DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/demos.json]: done

**Comment by topalach** [comment]
@Danielle9897 please resolve conflicts.

**Pull request merged**